src/Fpl.Client/GlobalSettingsClient.cs

```diff
@@ -1,27 +1,22 @@
-using System.Net.Http;
-using System.Threading.Tasks;
 using Fpl.Client.Abstractions;
 using Fpl.Client.Models;
-using Newtonsoft.Json;
+using System;
+using System.Threading.Tasks;
 
 namespace Fpl.Client
 {
     public class GlobalSettingsClient : IGlobalSettingsClient
     {
-        private readonly HttpClient _client;
+        private readonly ICachedHttpClient _client;
 
-        public GlobalSettingsClient(HttpClient client)
+        public GlobalSettingsClient(ICachedHttpClient client)
         {
             _client = client;
         }
 
-        public async Task<GlobalSettings> GetGlobalSettings()
+        public Task<GlobalSettings> GetGlobalSettings()
         {
-            var json = await _client.GetStringAsync("/api/bootstrap-static/");
-
-            var data = JsonConvert.DeserializeObject<GlobalSettings>(json);
-
-            return data;
+            return _client.GetCachedOrFetch<GlobalSettings>("/api/bootstrap-static/", TimeSpan.FromHours(24));
```

Comment:
Tror cache-headern de sender sier 5 minutter her. 24 timer tror jeg vil brekke andre features, da vi poller gameweeks for status på gw-overganger og sender feks standings onGameweekFinished. Er ikke så bekymra for å kalle global-settings-endepunktet. De bruker en Varnish-cache og output-cacher det ganske heftig.

Reply:
Satte denne ned til 5 min for å matche cache-headere i 825d573